lib/backports/2.6.0/hash/merge.rb

```diff
@@ -3,8 +3,8 @@
 class Hash
   unless instance_method(:merge).arity < 0
     def merge_with_backports(first = {}, *others, &block)
-      merge_without_backports(first, &block).
-        merge!(*others, &block)
+      merge_without_backports(first, &block)
+        .merge!(*others, &block)
```

Comment:
This won't work in older Rubies... Again shouldn't be a problem in the future.

Reply:
Oh, my bad. When DotPosition cop reported only one offense, I thought it is because everywhere else in codebase it is leading dot, but turns out it is just the only multi-line chain :)